scripts/apps/authoring/views/authoring-header.html

```diff
@@ -4,49 +4,68 @@
             <i class="filetype-icon-{{ :: item.type}}" title="{{'Article Type'| translate }}: {{ item.type}}"></i>
         </div>
         <div ng-if="item.profile && !change_profile">
-            <label translate>PROFILE</label>
-            <div class="data" sd-item-profile data-profile="item.profile"></div>
+            <span class="authoring-header__label" translate>PROFILE</span>
+            <div class="authoring-header__value" sd-item-profile data-profile="item.profile"></div>
         </div>
         <div ng-if="change_profile && content_types.length">
-            <label translate>PROFILE</label>
-            <div class="data">
+            <span class="authoring-header__label" translate>PROFILE</span>
+            <div class="authoring-header__value" >
                 <select id="item-profile"
                         ng-model="item.profile"
                         ng-change="changeProfile(item)"
+                        style="padding: 0px; height: 16px; font-size: 11px;"
                         ng-options="type._id as type.label for type in desk_content_types">
                 </select>
             </div>
         </div>
         <div ng-if="item.type === 'text'">
-            <label class="word-count"><b sd-word-count data-item="item.body_html" data-html="true" data-count-only="true"></b> <span translate translate-n="item.word_count" translate-plural="WORDS">WORD</span></label>
+            <span class="authoring-header__value">
+                <strong sd-word-count data-item="item.body_html" data-html="true" data-count-only="true"></strong>
+            </span>
+            <span class="authoring-header__label" translate translate-n="item.word_count" translate-plural="WORDS">WORD</span>
         </div>
         <div ng-if="item.signal">
             <span class="signal">{{ item.signal}}</span>
         </div>
         <div>
-            <label translate>SOURCE</label>
-            <div id="item-source" class="data">{{ item.source}}</div>
+            <span class="authoring-header__label" translate>SOURCE</span>
+            <div class="authoring-header__value" id="item-source" >{{ item.source}}</div>
         </div>
-        <div ng-if="relatedItems._items.length">
-            <label translate>RELATED</label>
-            <div class="data"><a href="" ng-click="activateWidget()">{{relatedItems._items.length}}</a></div>
+        <div ng-if="relatedItems != null && Array.isArray(relatedItems._items) && relatedItems._items.length > 0">
```

Comment:
angular should handle all those error states so checking for `.length` being `> 0` or just `true` value is enough and more readable imo